test/resources/Account.tests.ts

```diff
@@ -1,65 +1,926 @@
 
-      (accounts_create_ach_response !== null).should.be.true;
+      const expect_results = {
+        id: accounts_create_ach_response?.id,
+        holder_id: holder_1_response?.id,
+        type: 'ach',
+        ach: { routing: '062103000', number: '123456789', type: 'checking' },
+        latest_verification_session: accounts_create_ach_response?.latest_verification_session,
+        products: [ 'payment' ],
+        restricted_products: [],
+        status: 'active',
+        error: null,
+        metadata: null,
+        created_at: accounts_create_ach_response?.created_at,
+        updated_at: accounts_create_ach_response?.updated_at
+      };
+
+      accounts_create_ach_response.should.be.eql(expect_results);
     });
 
     it('should successfully create a `liability` account.', async () => {
       accounts_create_liability_response = await client.accounts.create({
-        holder_id: holder_1_response.id,
+        holder_id: holder_1_response?.id || '',
         liability: {
-          mch_id: 'mch_3',
-          account_number: '123456789',
+          mch_id: 'mch_302086',
+          account_number: '4936494462408721',
         },
       });
 
-      (accounts_create_liability_response !== null).should.be.true;
+      accounts_create_liability_response.products.sort();
+
+      const expect_results = {
+        id: accounts_create_liability_response?.id,
+        holder_id: holder_1_response?.id,
+        type: 'liability',
+        liability: {
+          fingerprint: null,
+          mch_id: 'mch_183',
+          mask: '8721',
+          ownership: 'unknown',
+          type: 'credit_card',
+          name: 'Chase Sapphire Reserve'
+        },
+        latest_verification_session: accounts_create_liability_response.latest_verification_session,
+        balance: null,
+        update: null,
+        card_brand: null,
+        products: [ 'balance', 'payment', 'sensitive', 'update' ].sort(),
+        restricted_products: accounts_create_liability_response?.restricted_products,
+        subscriptions: accounts_create_liability_response.subscriptions,
+        available_subscriptions: [ 'update' ],
+        restricted_subscriptions: [],
+        status: 'active',
+        error: null,
+        metadata: null,
+        created_at: accounts_create_liability_response?.created_at,
+        updated_at: accounts_create_liability_response?.updated_at
+      };
+
+      const accounts_create_liability_response_sorted = {
+        ...accounts_create_liability_response,
+        products: accounts_create_liability_response?.products?.sort(),
+      };
+
+      accounts_create_liability_response_sorted.should.be.eql(expect_results);
     });
   });
 
   describe('accounts.get', () => {
     it('should successfully get an account.', async () => {
-      accounts_get_response = await client.accounts.get(accounts_create_ach_response.id);
+      accounts_get_response = await client.accounts.retrieve(accounts_create_ach_response?.id || '12345');
```

Comment:
Should tests be adjusted so `accounts_create_ach_response` is always non-null? Or is there a reason this test file has been changed to sometimes not expect objects to exist

Reply:
this is because we initialize these var's as `null`, and I added the `"strictNullChecks": true` to `tsconfig.json`, so TS wants the check there to make sure its non-null

Reply:
changed this so that we no longer initialize as `null`